ios/RCTOneSignal/RCTOneSignalEventEmitter.m

```diff
@@ -351,7 +351,7 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
     [OneSignal removeExternalUserId];
 }
 
-RCT_EXPORT_METHOD(didSetNotificationOpenedHandler) {
```

Comment:
We need both `initNotificationOpenedHandlerParams` and `initInAppMessageClickHandlerParams` stubbed for iOS since these can both be called from javascript